assets/sass/_langchoose.scss

```diff
@@ -117,6 +117,21 @@ $tab-font-size: 14px !default;
     cursor: pointer;
 }
 
+// Remove the top margin from code snippets under lang chooser tabs so the tabs sit directly on top.
+.langtab-tabs + .clearfix + .highlight,
+.langtab-tabs + .clearfix + .highlight + .highlight,
+.langtab-tabs + .clearfix + .highlight + .highlight + .highlight,
+.langtab-tabs + .clearfix + .highlight + .highlight + .highlight + .highlight,
```

Comment:
This is all pretty gross/lame -- but I couldn't think of a better way to select each of the possible code snippets under the lang tabs. Open to feedback if there's a better way to do it.

Reply:
Ah! This is a job for the [general sibling combinator](https://developer.mozilla.org/en-US/docs/Web/CSS/General_sibling_combinator):

```
.mdl-tabs {
    ...

    ~ .highlight > .chroma {
        margin-top: 0;
    }
}
```

... seems to do the trick.

Reply:
... although, I wonder if that messes up anything _below_ the language chooser. 🤔 Probably does.

Reply:
Thanks! TIL 😄. From poking around the site, it seems to work OK and not disrupt anything below the language chooser.